pkg/cloudprovider/backup_service.go

```diff
@@ -194,8 +194,15 @@ func (br *backupService) DeleteBackupDir(bucket, backupName string) error {
 	return errors.NewAggregate(errs)
 }
 
-func (br *backupService) CreateBackupLogSignedURL(bucket, backupName string, ttl time.Duration) (string, error) {
-	return br.objectStorage.CreateSignedURL(bucket, getBackupLogKey(backupName), ttl)
+func (br *backupService) CreateBackupSignedURL(backupType api.DownloadTargetKind, bucket, backupName string, ttl time.Duration) (string, error) {
```

Comment:
I'm working on adding restore log support, and I'm planning on having the restore log file name have the format `restore-<restore name>-logs.gz` e.g. `restore-mybackup-20170912150214-logs.gz`. I suppose after we merge this PR, we could rename this to just `CreateSignedURL`. WDYT?